docs/UserGuide.md

```diff
@@ -117,43 +117,22 @@ Adds a contact to the address book.
 
 Format: `add n/NAME p/PHONE e/EMAIL a/ADDRESS [t/TELEGRAM_USERNAME] [r/ROLE]…​`
 
+* Telegram usernames must start with an alphabet, contain only alphanumeric characters or underscores and be between 5 and 32 characters long.
+* The roles must be one of the following: attendee, vendor, volunteer and sponsor.
+
 <div markdown="span" class="alert alert-primary">:memo: **Note:**
 A contact can have any number of roles (including 0), and the Telegram username is optional. 
-The roles must be one of the following: attendee, vendor, volunteer and sponsor.
-Telegram usernames must start with an alphabet.
+
+:memo: **Note:** You will be unable to add a contact to PlanPal if there already exists a contact with the same phone number, email or telegram username.
```

Comment:
This is useful and addresses the ambiguity surrounding `Person` equality in our app!